frontend/src/stores/levels/actions.ts

```diff
@@ -0,0 +1,34 @@
+import { levels } from './state'
+import { get } from '@/services/http'
+
+export interface Level {
```

Comment:
This interface already exists under domain folder. Please, get rid of it, and use the interface of domain (if it doesn't exist, create like this)